src/components/desktop/Header.js

```diff
@@ -48,28 +76,89 @@ const StyledHeader = styled.header`
           font-family: inherit;
           font-weight: normal;
           cursor: pointer;
+          background-color: ${({ theme }) => theme.background};
         }
 
         button,
         &.active {
-          background-color: ${({ theme }) => theme.background};
           color: ${({ theme }) => theme.orange};
         }
       }
     }
   }
 `
+const Dropdown = ({ className, children, text }) => (
+  <div className={className}>
+    <div className="dropbtn">
+      {text} <CaretDown height="8px"> </CaretDown>
+    </div>
+    <div className="dropdown-content">{children}</div>
```

Comment:
One nice thing about styled-components is that our [class names are scoped to each component](https://styled-components.com/docs/api#:~:text=Normally%2C%20styled%20components%20are%20automatically%20scoped%20to%20a%20local%20CSS%20class%20and%20therefore%20isolated%20from%20other%20components.). It might cleaner to call this class `content` (or maybe something even more descriptive!) and similarly referring to `dropbtn` as `button`